ktxspec.adoc

```diff
@@ -1085,24 +1090,25 @@ Premultiplied Alpha::
   field if the images' RGB components have been multiplied by their
   alpha components, otherwise it must be 0.
 
-Basis Universal UASTC Format::
+Basis Universal ASTC Format::
```

Comment:
I think this change is not needed. "Basis Universal" is the "vendor" name and UASTC is the format name so the "universal" word should be present twice.

For comparison, the ETC1S section is called "Basis Universal ETC1S Format".

Reply:
Nothing to do with @lexaknyazev's point but @richgel999 are you happy for us to continue using _Basis Universal_ as the "vendor" name in these sections?

Reply:
> Nothing to do with @lexaknyazev's point but @richgel999 are you happy for us to continue using _Basis Universal_ as the "vendor" name in these sections?

Yes, that's fine.